src/core/game.ts

```diff
@@ -96,25 +97,45 @@ export class Game {
       Object.assign(list, (game.entities as any)[key]);
       return list;
     }, {});
+    let { walls, players }  = game.entities;
 
     events.forEach(event => {
       let sender = allEntities[event.initiator];
       let receiver = allEntities[event.receptor];
       switch(event.type) {
-        case 'COLLISION':
+        case 'COLLISION': 
           switch(sender.type) {
             case 'player': Player.collideWith(sender as PlayerState, receiver, game); break;
             case 'bullet': Bullet.collideWith(sender as BulletState, receiver, game); break;
           }
           break;
-        case 'SPAWN_BULLET':
+        case 'SPAWN_BULLET': 
           switch(sender.type) {
             case 'player':
               let bullet = Bullet.spawnFrom(sender as PlayerState);
               game.entities.bullets[bullet.id] = bullet;
               break;
           }
           break;
+        case 'MOVEMENT': 
+          if (sender) {
+            let movementData = event.data as PlayerMovement; 
+            Player.move(sender as PlayerState, movementData.angle, movementData.xVel, movementData.yVel);
+            let foundCollision = _.find(walls, (value : any, key : any) => {
```

Comment:
```js
let foundCollision = !!_.find(walls, wall => Entity.colliding(sender, wall))
                  || !!_.find(players, (player, playerId) => Entity.colliding(sender, player) && sender.id !== playerId);
```

`!!` is the Javascript way of turning a truthy/falsey value into `true` or `false`